brianlong/www.validators.app
# 183393419 group map results

#### What's this PR do?
- Adds an option to toggle nodes visibility (validators should always be visible)
- Groups map points into larger clusters

#### How should this be manually tested?
- If you have some data centers locally, you can skip this part. If you don't, you need to generate some localization data first. Turn on sidekiq and run:
```
rails r script/data_centers_scripts/append_data_centers_geo_data.rb
rails runner "DataCenterStatsWorker.perform_async('mainnet')"
```
or test on staging
- Go to /validators-map page and review. Click on nodes visibility button, click on points etc. Most of DCs should be grouped by country, DCs in US should be grouped by state.

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/n/projects/2177859/stories/183393419)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
